File: Problem Solving/Algorithms/Warmup/Plus Minus.c
void plusMinus(int arr_count, int* arr)
{
    float positive = 0, negative = 0, zeros = 0;
    
    for (int i = 0; i < arr_count; i++)
    {
        if (arr[i] > 0)
        {
            positive++;
        }
        else if (arr[i] < 0)
        {
            negative++;
        }
        else
        {
            zeros++;
        }
    }
    
    printf("%f\n%f\n%f", positive / arr_count, negative / arr_count, zeros / arr_count);
}

File: Problem Solving/Algorithms/Warmup/Plus Minus.cs
using System;
using System.Linq;

class Solution
{
    void plusMinus(int[] arr)
    {
        var total = arr.Count();
        
        Console.WriteLine(((double)arr.Count(x => x > 0) / total).ToString("N6"));
        
        Console.WriteLine(((double)arr.Count(x => x < 0) / total).ToString("N6"));
        
        Console.WriteLine(((double)arr.Count(x => x == 0) / total).ToString("N6"));
    }
}
